cmd/manager/main.go

```diff
@@ -1,180 +0,0 @@
-package main
-
-import (
-	"context"
-	"flag"
-	"fmt"
-	"os"
-	"runtime"
-	"strings"
-	"time"
-
-	"github.com/openshift/ocm-agent-operator/pkg/apis"
-	controllerconst "github.com/openshift/ocm-agent-operator/pkg/consts/controller"
-	"github.com/openshift/ocm-agent-operator/pkg/controller"
-	"github.com/openshift/ocm-agent-operator/pkg/localmetrics"
-	"github.com/openshift/ocm-agent-operator/version"
-	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
-
-	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
-	configv1 "github.com/openshift/api/config/v1"
-	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	"github.com/operator-framework/operator-sdk/pkg/leader"
-	"github.com/operator-framework/operator-sdk/pkg/log/zap"
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	"github.com/spf13/pflag"
-	"sigs.k8s.io/controller-runtime/pkg/cache"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
-
-	// OSD metrics
-	osdmetrics "github.com/openshift/operator-custom-metrics/pkg/metrics"
```

Comment:
Why was operator-custom-metrics dropped, is it not operator SDK v1.x compatible? There will be several implications for existing clusters with OAO installed that move to this version as a result.